subprojects/google-highway.wrap

```diff
@@ -0,0 +1,9 @@
+[wrap-file]
+directory = highway-1.2.0
+source_url = https://github.com/google/highway/releases/download/1.2.0/highway-1.2.0.tar.gz
+source_filename = highway-1.2.0.tar.gz
+source_hash = 58e9d5d41d6573ad15245ad76aec53a69499ca7480c092d899c4424812ed906f
+patch_directory = google-highway
+
+[provide]
+highway = google_highway_dep
```

Comment:
This doesn't look like it matches what cmake does, which is provide `hwy-config`. This should match (as should the name in the meson.build and the releases.json)